src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -83,6 +83,7 @@ public void ConfigureServices(IServiceCollection services)
     
             services.AddScoped(serviceProvider => new CertificateAuthenticationValidator(certificateAuthenticationConfig));
 #endif
+            services.AddRouting(options => options.LowercaseQueryStrings = true);
```

Comment:
I presume you mean `LowercaseUrls` instead of `LowercaseQueryStrings`?

If `LowercaseQueryStrings` uses camel casing I'd be happy to add that one as well

Reply:
OMG, haste makes waste.

I've added the LowercaseUrls settings and kept the LowercaseQueryStrings as well.  (The latter one can only be applied if the former one is set to true as well).

Reply:
What does a query parameter look like then - Is it `fooBar` or `foobar`?

Reply:
I would expect it to be `foobar` , but I've just tested it and it is `fooBar` (my query parameter is declared like that as well), so I'm confused now.

Reply:
lol that's odd. Do you want to keep it in then or not?

Reply:
I think we should keep it in

https://github.com/dotnet/aspnetcore/issues/10746